app/contributing/community.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Community
+description: Join the Kong Documentation Contribution Community to collaborate, learn, and contribute to open source projects. Start your journey with Kong today!
```

Comment:
```suggestion
purpose: Join the Kong Documentation Contribution Community to collaborate, learn, and contribute to open source projects. Start your journey with Kong today!
```
Does is need to be "description" exactly? In other docs, we've started using the "purpose" tag to do this.

Reply:
I know that the recognized HTML tag search engines use is <meta name="description" content="..."/> to identify the meta description that is to be shown in Google search results.  https://developers.google.com/search/docs/crawling-indexing/special-tags

Reply:
Ahh that's irrelevant for the tag you use here, then 😄 we generate the meta based on whatever arbitrary key we choose to set: (https://github.com/Kong/docs.konghq.com/blob/main/app/_includes/head.html#L48)

So either "description" or "purpose" would work, since we'd transform it into the right meta anyway. But we should use "purpose" for consistency.